llumnix/backends/vllm/llm_engine.py

```diff
@@ -244,14 +245,44 @@ def __init__(
         self._run_workers("init_migration", instance_id=instance_id, migration_config=migration_config,\
                                                       src_worker_handle_list=self.worker_handle_list,
                                                       placement_group=placement_group, node_id=node_id)
+
+        self.state_lock = threading.Lock()
+        self.state = EngineState.INIT
+        logger.info("{} current state {}".format(self.instance_id, self.state))
```

Comment:
Instance {} current state... might be better. Otherwise the instance_id might be confused with request_id. So is elsewhere.